types/synetica/enlinkStatusT/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Enlink T",
+  "name": "enLink Status-T",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed